configure.ac

```diff
@@ -358,6 +358,15 @@ AC_SUBST(LIBCRYPT)
 AC_CHECK_LIB(crypt, crypt, [LIBCRYPT=-lcrypt],
 	[AC_MSG_ERROR([crypt() not found])])
 
+AC_SUBST(LIBCAP)
+
+AC_CHECK_HEADERS(sys/capability.h,
+	[AC_CHECK_LIB(cap, capget, [
+		LIBCAP=-lcap
+		AC_DEFINE(HAVE_LIBCAP, 1, [Defined if you have libcap.])],
+		[LIBCAP=""])],
+	[LIBCAP=""])
```

Comment:
Hm, why do you even check for libcap if all you use are the two raw syscalls `cap{g,s}et()`? There's no need to link `new{g,u}idmap` against `libcap` imho.